include/scf/driver/commutator.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <simde/simde.hpp>
+
+namespace scf::driver {
+
+/** @brief Evaluates the commutator of matrix A, matrix B, with the Overlap S.
+ *
+ *  The commutator function evaluates the commutator between matrix A, matrix B,
+ *  and the Overlap matrix S. This requires that all of the matrices are square.
+ *
+ *  @tparam simde::type::tensor
```

Comment:
```suggestion
```

There is no template parameters. `@tparam` is for documenting template parameters.